src/plugin.js

```diff
@@ -261,6 +261,11 @@ const contribAdsPlugin = function(options) {
           player.ads.adTimeoutTimeout = window.setTimeout(function() {
             player.trigger('adtimeout');
           }, settings.prerollTimeout);
+          // We will try to pause content here. This is because in an autoadvance playlist
+          // on Android devices with HLS renditions, the duration is incorrectly reported
+          // as infinity, due to which the content is not paused incorrectly
+          // So before we signal to the ad plugin , we want to pause content
+          cancelContentPlay(player);
```

Comment:
Talking with Greg, since the playing of live video in playlist in the background is not a supported use-case, I'm going to go ahead with using shouldPlayContentBehindAd instead of cancelContentPlay as that would also address the concern of client side ads and live video in the background. 
We would also need to change or modify the adstart conditional check on Infinity duration in videojs-ima3.